### PR TITLE
Add changes for edge-22.9.1

### PR DESCRIPTION
**Pull request by jeremychase**
## edge-22.9.1

This release adds the `linkerd.io/trust-root-sha256` annotation to all injected
workloads allowing predictable comparison of all workloads' trust anchors via
the Kubernetes API.

Additionally, this release lowers the inbound connection pool idle timeout to
3s. This should help avoid socket errors, especially for Kubernetes probes.

* Added `linkerd.io/trust-root-sha256` annotation on all injected workloads
  to indicate certifcate bundle
* Lowered inbound connection pool idle timeout to 3s
* Restored `namespace` field in Linkerd helm charts
* Updated fields in `AuthorizationPolicy` and `MeshTLSAuthentication` to
  conform to specification (thanks @aatarasoff!)
* Updated the identity controller to not require a `ClusterRoleBinding`
  to read all deployment resources.

### CODE REVIEW

**Review by adleong — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changes
 
+## edge-22.9.1
+
+This release adds the `linkerd.io/trust-root-sha256` annotation to all injected
+workloads; allowing predictable comparison of all workloads' trust anchors via
+the Kubernetes API.
```
Comment (kleimkuhler) [inline CHANGES.md:7]: nit: or comma. As written, the second part is not a correct sentence, so the semicolon doesn't belong.

```suggestion
This release adds the `linkerd.io/trust-root-sha256` annotation to all injected
workloads allowing predictable comparison of all workloads' trust anchors via
the Kubernetes API.
```

**Comment by adleong** [comment]
Should the changenotes also note about lowering the inbound connection pool idle timeout to 3s?

**Review by kleimkuhler — APPROVED** [review]

**Comment by jeremychase** [comment]
@adleong @kleimkuhler Updated; thank you!

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+Additionally, this release lowers the inbound connection pool idle timeout to
+3s. This should help avoid socket errors, especially for Kubernetes probes.
+
+* Added `linkerd.io/trust-root-sha256` annotation on all injected workloads
+  to indicate certifcate bundle
+* Lowered inbound connection pool idle timeout to 3s
+* Restored `namespace` field in Linkerd helm charts
+* Updated fields in `AuthorizationPolicy` and `MeshTLSAuthentication` to
+  conform to specification (thanks @aatarasoff!)
+
```
Comment (olix0r) [inline CHANGES.md:18]:
```suggestion
* Update the identity controller to not require a `ClusterRoleBinding`
  to read all deployment resources.

```

**Pull request merged**